src/index.html

```diff
@@ -6,9 +6,90 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="top__page">
+      <div class="container">
+        <div class="logo">
+          <img
+          src="./images/logo.png"
+          alt="moyoLogo"
+          >
+        </div>
+        <nav class="nav">
+          <ul class="top__menu list">
+            <li class="top__menu__item">
+              <a
+                href="#Apple"
+                class="button is-active"
+              >
+              Apple
+              </a>
+            </li>
+            <li class="top__menu__item">
```

Comment:
Use nav, nav__list, nav__item, nav__link classes
```suggestion
        <nav class="nav">
          <ul class=" nav__list">
            <li class="nav__item">
              <a
                href="#Apple"
                class="nav__link is-active"
              >
              Apple
              </a>
            </li>
            <li class="top__menu__item">
```